x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -105,7 +105,7 @@
     public static final String MODEL_DEFINITION_NOT_FOUND = "Could not find trained model definition [{0}]";
     public static final String MODEL_METADATA_NOT_FOUND = "Could not find trained model metadata {0}";
     public static final String INFERENCE_CANNOT_DELETE_MODEL =
-        "Unable to delete model [{0}]";
+        "Unable to delete model [{0}] as it is self-contained in the machine learning plugin";
```

Comment:
@sophiec20 I am open to better suggestions here.

Reply:
The string `INFERENCE_CANNOT_DELETE_MODEL` is named generically as if the the message could apply to any model but it is strictly about ml models. 

I see it is only used in one place but can you rename to `INFERENCE_CANNOT_DELETE_*ML*_MODEL` or something pls. 


Suggestions:
"... as it is managed by machine learning"

Reply:
Suggest `Cannot delete built-in model [{0}] as it is required by machine learning`

Reply:
👍